src/application/boltz/claim.ts

```diff
@@ -0,0 +1,151 @@
+import type { OwnedInput } from 'liquidjs-lib';
+import {
+  Creator,
+  Updater,
+  Extractor,
+  Finalizer,
+  CreatorInput,
+  Transaction,
+  witnessStackToScriptWitness,
+  Pset,
+  ZKPValidator,
+  ZKPGenerator,
+  Blinder,
+  AssetHash,
+} from 'liquidjs-lib';
+import type { UnblindedOutput } from 'marina-provider';
+import type { Output } from 'liquidjs-lib/src/transaction';
+import { SignerService } from '../signer';
+import type { AppRepository, WalletRepository } from '../../domain/repository';
+import zkp from '@vulpemventures/secp256k1-zkp';
+
+/**
+ * Claim swaps
+ *
+ * @param utxos UTXOs that should be claimed or refunded
+ * @param witnessUtxo
+ * @param preimage
+ * @param redeemScript
+ * @param destinationScript the output script to which the funds should be sent
+ * @param fee how many satoshis should be paid as fee
+ * @param isRbf whether the transaction should signal full Replace-by-Fee
+ * @param assetHash asset hash of Liquid asset
+ * @param walletRepository
+ * @param appRepository
+ * @param password
+ * @param blindingKey blinding public key for the output; undefined if the output should not be blinded
+ * @param timeoutBlockHeight locktime of the transaction; only needed if the transaction is a refund
+ */
+export async function constructClaimTransaction(
+  utxos: UnblindedOutput[],
+  witnessUtxo: Output,
+  preimage: Buffer,
+  redeemScript: Buffer,
+  destinationScript: Buffer,
+  fee: number,
+  assetHash: string,
+  walletRepository: WalletRepository,
+  appRepository: AppRepository,
+  password: string,
+  isRbf = true,
```

Comment:
always called with `true`, is it necessary ?

Reply:
No idea